hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/subscription/SubscriptionService.java

```diff
@@ -120,6 +126,13 @@ public SubscriptionMetrics getSubscriptionMetrics(TopicName topicName, String su
         return metricsRepository.loadMetrics(topicName, subscriptionName);
     }
 
+    public SubscriptionHealth getSubscriptionHealth(TopicName topicName, String subscriptionName) {
+        Subscription subscription = getSubscriptionDetails(topicName, subscriptionName);
+        TopicMetrics topicMetrics = topicService.getTopicMetrics(topicName);
+        SubscriptionMetrics subscriptionMetrics = getSubscriptionMetrics(topicName, subscriptionName);
```

Comment:
maybe we should invest in some subscriptionmetrics cache now, those values won't change per request, but in time